roles/hardening/files/limits.conf

```diff
@@ -0,0 +1,58 @@
+# /etc/security/limits.conf
+#
+#Each line describes a limit for a user in the form:
```

Comment:
Won't changes to this file require a reboot?

Reply:
I think this is an ansible role to be consume by the baking of the AMI, so it will be set at boot time

Reply:
Can we get it into your ami builder then and I will remove it from my playbook

Reply:
and /etc/security/limits.conf will affect new sessions ( needing a reboot to make sure it catches everything)